.github/workflows/ci.yml

```
@@ -64,3 +65,26 @@ jobs:
           creds: ${{ secrets.GITHUB_TOKEN }}
           registry: ghcr.io
           image: ghcr.io/thermondo/sigsci
+
+  build-java:
+    runs-on: ubuntu-22.04
+    needs: lint
+    strategy:
+      matrix:
+        version:
+          - "21"
+    steps:
+      - uses: actions/checkout@v3
```

To match with the rest of the file in main, small update here:
```suggestion
      - uses: actions/checkout@v4
```